### ISSUE TITLE
Document using aria attributes with element used to toggle terra-toggle component

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#1920]
# Feature Request

## Description
Certain aria attributes should be used when expanding and collapsing the terra-toggle component. These need to be set on the element users are interacting with outside of the terra-toggle component that is triggering the expand and collapse. We should document these so they are clear for our users.

https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions
